README.md

```diff
@@ -87,6 +87,8 @@ Distributed under the Apache 2.0 License. See [LICENSE](LICENSE.txt) for more in
 <!-- CONTACT -->
 # Contact
 
+Discord - [Join the thin-edge.io community now!](https://discord.com/invite/SvqWp6nrsK)
```

Comment:
This links to an invalid invite.

Reply:
This link comes from here: https://github.com/thin-edge/community . A GitHub Project which does not seem to be maintained and should probably be deleted. Can you replace the link with a valid invite?

Reply:
This links is from Sebastian's user to invite a new user. We should change it to use either Didier, Reuben or one of the generic thin edge user which we use fot he bots.

Reply:
Although the link is still valid, probably better to create a new link as @itsroshan suggests

Reply:
@reubenmiller can you create a new invite link with no limit (no number of usage and no expiration date)

Reply:
Resolved by 9651f732c7d8baa946ebe26665686e4509feaf56